build.properties.default

```diff
@@ -286,14 +286,14 @@ saaj-api.home=${base.path}/saaj-api-${saaj-api.version}
 saaj-api.jar=${saaj-api.home}/saaj-api-${saaj-api.version}.jar
 saaj-api.loc=${base-maven.loc}/javax/xml/soap/saaj-api/${saaj-api.version}/saaj-api-${saaj-api.version}.jar
 
-# ----- bnd & bndlib, version 4.0.0 or later  -----
+# ----- bnd, version 5.1.1 or later  -----
 # ----- provides OSGI metadata for JARs       -----
-bnd.version=5.1.0
+bnd.version=5.1.1
 
 # checksums for biz.aQute.bnd-5.1.0.jar
```

Comment:
You need to update the version in the comment for consistency.

Reply:
Done!